internal/binarylog/sink.go

```diff
@@ -90,14 +90,17 @@ type bufferedSink struct {
 	out    Sink          // out is built on buf.
 	buf    *bufio.Writer // buf is kept for flush.
 
-	writeStartOnce sync.Once
+	flusherStarted bool
```

Comment:
Could you please group this new field with the above ones which are currently being protected by the mutex.

Reply:
Done